src/doc/unstable-book/src/compiler-flags/remap-path-scope.md

```diff
@@ -10,10 +10,8 @@ This flag accepts a comma-separated list of values and may be specified multiple
 
 - `macro` - apply remappings to the expansion of `std::file!()` macro. This is where paths in embedded panic messages come from
 - `diagnostics` - apply remappings to printed compiler diagnostics
-- `unsplit-debuginfo` - apply remappings to debug information only when they are written to compiled executables or libraries, but not when they are in split debuginfo files
-- `split-debuginfo` - apply remappings to debug information only when they are written to split debug information files, but not in compiled executables or libraries
-- `split-debuginfo-path` - apply remappings to the paths pointing to split debug information files. Does nothing when these files are not generated.
-- `object` - an alias for `macro,unsplit-debuginfo,split-debuginfo-path`. This ensures all paths in compiled executables or libraries are remapped, but not elsewhere.
+- `debuginfo` - apply remappings to debug informations
+- `object` - an alias for `macro,debuginfo`. This ensures all paths in compiled executables or libraries are remapped, but not elsewhere.
```

Comment:
Is it possible that in the future `object` evolves beyond an alias for `macro,debuginfo`? I may suggest this:

```suggestion
- `object` - This ensures all paths in compiled executables or libraries are remapped, but not elsewhere. Currently an alias for `macro,debuginfo`.
```

Reply:
I think that's a good suggestion.

Reply:
Agree, changed.